apps/01PigLatin.js

```diff
@@ -9,8 +9,42 @@ var rl = readline.createInterface({
 
 
 function pigLatin(word) {
-
+word=word.toLowerCase();
   // Your code here
```

Comment:
Try to always add spaces between infix operators like `=`, `+`, etc:

`word = word.toLowerCase();`